### PR TITLE
PP-6296 Whitelist nmni domain

### PR DESCRIPTION
**Pull request by sfount**
Allow users to sign up with `@nmni.com` emails as per PP-6926.

### CODE REVIEW

**Comment by rorymalcolm** [comment]
actually wait, we have tests for this

File: src/main/java/uk/gov/pay/adminusers/utils/email/EmailValidator.java [hunk 1]
```diff
@@ -44,7 +44,8 @@
             "socialworkengland\\.org\\.uk",
             "ucds\\.email",
             "wmca\\.org\\.uk",
-            "york\\.ac\\.uk"
+            "york\\.ac\\.uk",
+            "nmni\\.com"
```
Comment (stephencdaly) [inline src/main/java/uk/gov/pay/adminusers/utils/email/EmailValidator.java]: the domains are in alphabetical order so far, so would be good to keep it that way

**Comment by sfount** [comment]
Thanks @rorymalcolm @stephencdaly, I've moved it to be alphabetically sorted -- testing every single input into a method feels like we're testing Java `util.List` but I can add it if it's necessary

**Comment by rorymalcolm** [comment]
> Thanks @rorymalcolm @stephencdaly, I've moved it to be alphabetically sorted -- testing every single input into a method feels like we're testing Java `util.List` but I can add it if it's necessary

I agree but I had stuff knocked back on this before for not having them

**Comment by sfount** [comment]
Understood @rorymalcolm, added them 👍 https://github.com/alphagov/pay-adminusers/commit/f105e6a9b6845a7660c9f534fb40d67369088204

**Pull request merged**